sklearn/utils/_clibs_helpers.pyx

```diff
@@ -0,0 +1,32 @@
+cimport openmp
```

Comment:
Does this work even if we compile scikit-learn without the openmp flag?

Reply:
Good.